var/spack/repos/builtin/packages/intel/package.py

```diff
@@ -33,6 +33,8 @@ class Intel(IntelPackage):
 
     homepage = "https://software.intel.com/en-us/intel-parallel-studio-xe"
 
+    version('18.0.1-cluster', '9c007011e0e3fc72747b58756fbf01cd',
```

Comment:
is not it here already https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/intel-parallel-studio/package.py#L40 ?

Reply:
You are correct, Sorry, I started with the wrong package I guess.